python/ray/tests/test_usage_stats.py

```diff
@@ -301,6 +303,41 @@ def test_pg_stats():
         )
 
 
+def test_task_stats(reset_usage_stats):
+    @ray.remote
+    def foo():
+        pass
+
+    with ray.init(
+        _system_config={"metrics_report_interval_ms": 10},
```

Comment:
I guess it's more of this that controls how quickly data gets populated: 
https://github.com/ray-project/ray/blob/c88934950f8035e1ad2c3defc9a28aceb0dd2f12/src/ray/common/ray_config_def.h#L447-L450